examples/extended/src/integration-test/groovy/page/AbstractSecurityPage.groovy

```diff
@@ -9,10 +9,11 @@ abstract class AbstractSecurityPage extends Page {
 	}
 
 	protected boolean assertContentContains(String expected) {
-		assert $().text().contains(expected)
+		assert $().text().contains(expected) || $().text().contains(expected)
```

Comment:
Duplicate check is being performed.

```
assert $().text().contains(expected) || $().text().contains(expected)
```

should be

```
assert $().text().contains(expected)
```

Reply:
fixed in next commit..I thought I got that before I checked it in